internal/scripts/release/release_preparation.sh

```diff
@@ -88,12 +88,7 @@ print_manual_steps_hint() {
     echo -e "* Test if Yocto builds and runs with the current codebase"
     echo -e "* check if the new features are marked as done, e.g. README, ROADMAP, etc."
     echo -e "* grep for 'planned'"
-    echo -e "* verify to be on the right branch"
-}
-
-print_check_code_examples() {
-    echo -e "* '\$GIT_ROOT$/README.MD'"
-    echo -e "* '\$GIT_ROOT$/internal/cpp_doc_generator/*.rst'"
```

Comment:
Is this completely gone, or just moved somewhere else?

Reply:
It is completely gone since we no longer have code examples at those locations. Everything moved to the iceoryx2 book.